code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -329,7 +329,8 @@
 					msg += "[t_He] [t_has] a stupid expression on [t_his] face.\n"
 		if(getorgan(/obj/item/organ/internal/brain))
 			if(ai_controller?.ai_status == AI_STATUS_ON)
```

Comment:
It looks like it's intentional for AI controllers to not look like themselves?

Reply:
In shot, this mechanic was intended for player species, and monkeys are (predominantly) an npc species which this code unintentionally applies to, do to a refractor in 2020.

Reply:
It is intentional. But not for monkeys.